### PR TITLE
7.x 1.x

### PR DESCRIPTION
**Pull request by AlexBrown-appnovation**
- basic html.tpl.php page added with some customization
- added meta tags found on web design standards site (mostly to do with icons and mobile optimizations)
- also includes scripts for IE9 and below at the bottom of the page just inside the end of the body
- updated info file with the same description as the d8 branch

### CODE REVIEW

File: templates/system/html.tpl.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <!-- Mobile Specific Metas -->
+    <meta name="HandheldFriendly" content="True">
+    <meta name="MobileOptimized" content="320">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Favicons -->
+    <!-- 128x128 -->
+    <link rel="icon" type="image/png" href="<?php print $theme_path; ?>/assets/img/favicons/favicon.png" />
+    <!-- 192x192, as recommended for Android http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android -->
+    <link rel="icon" type="image/png" sizes="192x192" href="<?php print $theme_path; ?>/assets/img/favicons/favicon-192.png" />
```
Comment (fureigh) [inline templates/system/html.tpl.php:20]: @AlexBrown-appnovation Great to see your work on this! Can you (create and) include the referenced image assets in this PR so the hrefs will work?

**Comment by fureigh** [comment]
@AlexBrown-appnovation I left an inline comment, but if this is a work-in-progress branch and you'd rather I hold off, just let me know.

**Comment by AlexBrown-appnovation** [comment]
@fureigh the image assets should already be in this branch in the assets folder. Let me know if I'm not understanding your request properly though!

**Comment by fureigh** [comment]
@AlexBrown-appnovation I see these files:
- `favicon.ico`
- `screenshot.png`

...but not these:
- `favicon-72.png`
- `favicon-114.png`
- `favicon-144.png`

...which are referenced in lines 24, 26 and 28 of `templates/system/html.tpl.php`. If I'm misunderstanding something, let me know.

**Comment by AlexBrown-appnovation** [comment]
@fureigh if you look in /assets/img/favicons/ you'll find everything in there.

**Comment by fureigh** [comment]
@AlexBrown-appnovation Beautiful, thanks! Will you add them to this PR?

**Comment by fureigh** [comment]
@AlexBrown-appnovation I can do it if you'd prefer, but I don't want to step on your toes. Also, it's usually safer to have the person who made something be the person who adds that thing.

**Comment by AlexBrown-appnovation** [comment]
Unless I'm mistaken, I believe the assets folder already exists in this branch! Must have been pushed up with one of my first commits!

**Comment by fureigh** [comment]
@AlexBrown-appnovation D'oh. You're right! Included with the USWDS assets folder. Merging away.

**Pull request merged**